modules/backend/widgets/Lists.php

```diff
@@ -709,6 +710,10 @@ public function getColumns()
      */
     public function getColumn($column)
     {
+        if (!isset($this->allColumns[$column])) {
+            throw new ApplicationException('No definition for column ' . $column);
+        }
```

Comment:
Could you provide some more background on the inclusion of this?

Reply:
I think that was added by @bennothommo

Reply:
@LukeTowers I noticed that filters had an exception for a missing scope - see here:
https://github.com/octobercms/october/blob/33f5b3933f99c6dd6da93ab3bdd833d86dde317e/modules/backend/widgets/Filter.php#L890-L892
I copied the same idea to Lists to make it easy to test, however, it is arguably a breaking change. We could potentially make that method and the one in Lists return `null` if a scope/column does not exist instead.

Reply:
I'm fine with leaving it in, was just curious on the background. Not really a breaking change because it would have thrown an error previously when trying to access an index that didn't exist.